src/components/document/annotation-layer.tsx

```diff
@@ -16,14 +16,14 @@ import { DocumentContentModelType } from "../../models/document/document-content
 import "./annotation-layer.scss";
 
 interface IAnnotationLayerProps {
+  canvasRef?: HTMLDivElement | null;
```

Comment:
I'd recommend calling this `canvasElt` or `canvasElement`, since a 'ref' implies it is a React ref which we'd need to call `.current` on to get the actual element.